app/controllers/comments.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime
+from flask import abort, current_app, g, jsonify, request
+from flask.views import MethodView
+from werkzeug.exceptions import BadRequestKeyError
+
+from app.controllers.controller import auth_token_required
+
+
+class Comments(MethodView):
+
+    decorators = [auth_token_required]
+
+    def get(self, task_id):
+        if not Comments.is_assigned(current_app.user.id, task_id):
+            abort(403, 'Forbidden')
+        cursor = g.connection.cursor()
+        query = 'SELECT user_id, text FROM comments WHERE task_id=%s'
+        cursor.execute(query, (task_id, ))
+        column_names = [x[0] for x in cursor.description]
+        data = []
+        for item in cursor.fetchall():
+            data.append(dict(zip(column_names, item)))
+        return jsonify(data)
+
+    def post(self, task_id):
+        if request.mimetype in ('multipart/form-data',
+                                'application/x-www-form-urlencoded'):
+            try:
+                comment = request.form['text']
+            except BadRequestKeyError:
+                abort(400, 'not valid data')
```

Comment:
errors should be consistent in scope of app, please check error format in case I haven't sent required fields for tasks creation.